test/index.test.js

```diff
@@ -1,28 +1,23 @@
-const assert = require('node:assert')
+const assert = require('node:assert/strict')
 const fs = require('node:fs/promises')
 const path = require('node:path')
-
+const {test} = require('node:test')
 const {commands, extensions, window, workspace} = require('vscode')
 
 module.exports.run = async () => {
```

Comment:
does this still need to be wrapped in `module.exports.run`?
Could the tests be run at the top level?

Reply:
Yes. This is how `@vscode/test-electron` works. It references a CJS module that should have a `run()` function that returns a promise. I.e. using another framework means this function should call something like `new Mocha()` or `jest.run()`.